Source/WebCore/platform/network/cocoa/NetworkStorageSessionCocoa.mm

```diff
@@ -420,15 +420,15 @@ - (CFURLRef)_cfurl;
         return Vector<Cookie> { };
 
     Vector<Cookie> cookiesVector;
-    NSString *name = options.name;
+    RetainPtr<NSString> name = static_cast<NSString *>(options.name);
```

Comment:
```suggestion
    RetainPtr name = options.name.protectedNSString();
```
While it’s not great to call a “protected” function when also putting something into a RetainPtr, I think it’s worse to write a cast rather than a named function. The reason we have the `operator NSString *` is so we can write things in a terse way, but a named function is clearer when we have to invoke it explicitly.

Reply:
How about I add a `nsString()` getter on String and call it? I'd prefer not to set an example initializing a local variable with a static function (though I hesitated when I wrote this PR).

Reply:
Sure, I would name it `createNSString()` just like `createCFString()`. But I’m not sure what you mean, what is the static function?

Reply:
Sorry, I meant a `protected` function, not a `static` function.

Reply:
I will use createNSString() and createNSURL()